src/Apps/CollectorProfile/Routes/Saves2/Components/SelectListsForArtworkModal/SelectListsForArtworkHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button, Flex, Spacer, Text } from "@artsy/palette"
+
+export const SelectListsForArtworkHeader = () => {
+  return (
+    <Flex flexDirection="row" alignItems="center">
+      <Flex flex={1} flexDirection="row" alignItems="center">
+        <Flex flexShrink={0} width={50} height={50} bg="black10" />
+        <Spacer x={1} />
+        <Text>Marie Pop, The NAR, 2016</Text>
```

Comment:
**question:** Intentional placeholder for now, yes?

Reply:
Yes, that's right. this will be changed as part of the [FX-4590](https://artsyproduct.atlassian.net/browse/FX-4590) task